### ISSUE TITLE
text-util.js toUpperCase() error

### ISSUE THREAD

**Issue by adamlundrigan** [orocrm/platform#529]
I have a grid column that displays multiple values concatenated together (using Twig), so I've titled the column "# Regs (R / WL / P)".  In 1.10.2 and 1.10.3 (and maybe 1.10.1...didn't roll back to try) I now get a javascript error when the datagrid is displayed:

```
Uncaught TypeError: Cannot read property 'toUpperCase' of undefined
```

The source of the problem is here:  
https://github.com/orocrm/platform/blob/4c2271ce5c7bb37745bcc5dfd6083731a9741b67/src/Oro/Bundle/UIBundle/Resources/public/js/tools/text-util.js#L84

It looks like when the string is tokenized to count it's length the non-alphanumeric characters are stripped but the resulting empty strings in the `words` array are not removed, so `words` looks like this:

```
["", "Regs", "R", "WL", "P", ""]
```

when the referenced line of JS tries to uppercase the first character there isn't one.

**Comment by Grygir** [comment]
Thanks for bug report. We will fix it soon.

**Comment by adamlundrigan** [comment]
Looks like it's been fixed in 1.10.5 :+1: 

https://github.com/orocrm/platform/commit/8117e099a1de0c15fd20bf8a6a4b4e9bc34bb02b#diff-a0b8d9e31e487faca2e522cdf6379e7b

**Comment by adamlundrigan** [comment]
Looks like it's been fixed in 1.10.5 :+1: 
https://github.com/orocrm/platform/commit/8117e099a1de0c15fd20bf8a6a4b4e9bc34bb02b#diff-a0b8d9e31e487faca2e522cdf6379e7b

**Comment by Grygir** [comment]
Yes it is.
